### PR TITLE
chore: remove debuggers dependency on core/package.json

### PR DESCRIPTION
**Pull request by hayemaxi**
Generate debuggers definitions file from toolkit. We will now store this in the repo.

There is a lot of code that relies on the definitions, which is why it is is still generated into core. Future work: drop this code down into packages/toolkit

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/toolkit/scripts/build/generateConfigurationAttributes.ts [hunk 1]
```diff
@@ -49,6 +38,31 @@ async function generateConfigurationAttributes(): Promise<void> {
...
             continue
         }
+
+        if (!fs.existsSync(path.dirname(debuggerConfig.outputFile))) {
+            throw new Error(`Output folder of ${debuggerConfig.outputFile} does not exist.`)
+        }
+
+        const header = `
+/*!
+* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (nkomonen-amazon) [inline packages/toolkit/scripts/build/generateConfigurationAttributes.ts:48]: Should we make this copyright blob a global constant? We'll probably create other generated files in the future. Maybe in a `scripts/constants.ts`

Comment (hayemaxi) [reply packages/toolkit/scripts/build/generateConfigurationAttributes.ts:48]: good idea 👍

**Review by justinmk3 — APPROVED** [review]
Great choices.

File: packages/toolkit/scripts/build/generateConfigurationAttributes.ts (hunk 1 above)
Comment (hayemaxi) [reply packages/toolkit/scripts/build/generateConfigurationAttributes.ts:48]: will create in a follow-up

**Pull request merged**